examples/ipyradiant.ipynb

```diff
@@ -86,6 +95,15 @@
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
    "version": "3.7.8"
+  },
+  "pycharm": {
```

Comment:
don't check this cruft in: we should probably add stripping this to nblint. can be a separate issue.